app/Resources/assets/js/lsdoc/index.js

```diff
@@ -1,32 +1,44 @@
 $(document).on('ready', function(){
     SaltGithub.init();
+    Dropdowns.init();
+
     $('input[name="import"]').click(function(){
         loadContent($('input[name="import"]:checked').val());
     });
     $('.import-framework').click(function(){
         ImportFrameworks.fromAsn();
     });
+    $('.send-info').click(function(){
+        if( Dropdowns.validDropdown("form.matched-fields-cfdof") &&
+            Dropdowns.validDropdown("form.matched-fields-cfitem") &&
+              Dropdowns.validDropdown("form.matched-fields-cfassociation")){
+            Import.send();
+        }
+    });
 });
 
 var SaltGithub = (function(){
 
     function init(){
-        getRepoList();
+        getRepoList(1, 30);
         $('#js-wizard-btn').click(function(){
             loadModal();
         });
     }
 
-    function getRepoList(){
+    function getRepoList(page, perPage){
         if( $('.js-github-list').length > 0 ){
-            $.get('/user/github/repos', function(data){
+            $.get('/app_dev.php/user/github/repos', { page: page, perPage: perPage }, function(data){
```

Comment:
We will want to ensure that all /app_dev.php/ URLs are removed once the code is ready.

We might want to use a javascript variable here that is set on the page where it is used so that it can automatically use either the dev or non-dev path as appropriate.